src/components/macroview/centerPanel/sortableElement/SortableItem.tsx

```diff
@@ -101,27 +96,28 @@ export default function SortableItem({
     if (checkIfElementIsEditable(element)) updateSelectedElementId(id - 1)
   }, [element, id, isSelected, recording, updateSelectedElementId])
 
-  const onEditButtonPress = useCallback(() => {
-    stopRecording()
-    if (isSelected) {
-      return
-    }
-    updateSelectedElementId(id - 1)
-  }, [id, isSelected, stopRecording, updateSelectedElementId])
-
   const onDeleteButtonPress = useCallback(() => {
     if (isSelected) {
       updateSelectedElementId(undefined)
     }
     onElementDelete(id - 1)
   }, [id, isSelected, onElementDelete, updateSelectedElementId])
 
+  const secondBg = useColorModeValue('primary-light.0', 'primary-dark.800')
+
   return (
     <HStack
       w="full"
       h="full"
+      bg={
```

Comment:
Tweaks for bg colors and other rounded corners for the sequencing area elements (we removed the image background in 1.1)